operations/route_handler.rb

```diff
@@ -0,0 +1,15 @@
+module RouteHandler
+  def self.build_route_view_object(route, params)
+    {
+        route_id:           route.id,
+        departure_station:  City.find(params[:departure_id]).title,
```

Comment:
Use `find_by` & safe navigational operator to avoid `RecordNotFound` errors, what if `City.find(...)` returns `nil`?